extensions/amp-facebook-comments/0.1/validator-amp-facebook-comments.protoascii

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-facebook-comments
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-facebook-comments extension .js script"
+  mandatory_parent: "HEAD"
+  unique_warning: true
+  satisfies: "amp-facebook-comments extension .js script"
+  requires: "amp-facebook-comments"
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    mandatory: true
+    value: "amp-facebook-comments"
+    dispatch_key: true
+  }
+  attrs: { name: "nonce" }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\\.ampproject\\.org/v0/amp-facebook-comments-(latest|0\\.1).js"
+  }
+  attrs: {
+    name: "type"
+    value_casei: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-facebook-comments.html"
+}
+tags: {  # <amp-facebook-comments>
+  html_format: AMP
+  tag_name: "AMP-FACEBOOK-COMMENTS"
+  disallowed_ancestor: "AMP-SIDEBAR"
+  satisfies: "amp-facebook-comments"
+  requires: "amp-facebook-comments extension .js script"
+  # data-* is generally allowed, but it's not generally mandatory.
+  attrs: {
+    name: "data-href"
+    mandatory: true
```

Comment:
/cc @ymschaap: this is good, but it just occurred to me, should we just default `href` to this page? Or canonical URL? Seems like most of time comments would be about the current page.